raiden/tests/unit/test_pfs_integration.py

```diff
@@ -705,6 +688,7 @@ def query_paths_args(token_network_state, our_address, pfs_max_fee):
         pathfinding_max_fee=pfs_max_fee,
         pathfinding_max_paths=3,
         pathfinding_iou_timeout=500,
+        pathfinding_fee=int(pfs_max_fee / 2),
```

Comment:
Any special reason for setting this to `pfs_max_fee / 2` and not something like `5`?

Reply:
I think we can stay in the same range as in `pfs_max_fee` that way, so in the futuer we might test with realistic prices like 10^12 tokens or so